coalib/bearlib/languages/definitions/C.py

```diff
@@ -0,0 +1,21 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class C:
+    extensions = '.c', '.h'
+    comment_delimiter = '//'
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'"': '"'}
```

Comment:
chars != strings

they're used to find strings in your source code for whatever reason a bear wantt'